app/src/main/java/com/saucelabs/mydemoapp/android/view/fragments/LoginFragment.java

```diff
@@ -148,12 +147,10 @@ public void onAuthenticationSucceeded(@NonNull BiometricPrompt.AuthenticationRes
                 super.onAuthenticationSucceeded(result);
                 Log.e(TAG, "onAuthenticationSucceeded: ");
 
-                fetchWiki();
-
                 TestFairy.setUserId(getMockBiometricUserName());
                 TestFairy.addEvent("User signed in with biometrics");
 
-//                ST.isLogin = true;
+                ST.isLogin = true;
```

Comment:
I'm not familiar with this change. What would this do?

Reply:
This change will set the state of the App to be `Authenticated`.
It will mainly replace the `Log In` button by the `Log Out` one.

`Log Out` allows to quit the session with using displaying visual defects.